gpio/README.md

```diff
@@ -120,8 +120,9 @@ Publishes to a MQTT connection with name `MQTT`:
 SensorTempOutside:
     Class: gpio.ds18x20_sensor.Ds18x20Sensor
     Connections:
-      MQTT:
-        StateDest: temp/outside
+        MQTT:
+            Temperature:
+                StateDest: temp/outside
```

Comment:
For future reference: 
I deliberately designed the Homie support that way, that the config for a sensor with single output doesn't need to specify the channel. So sensorReporter will assume that the given `StateDest` is reverencing the only available channel.
So, in my opinion, both configurations (with and without channel) are correct in this case.

Reply:
The original/old configuration did not work due to the MQTT Homie changes, so if both should, we need to fix this.

Reply:
Both should I think. Not everyone is going to want to use Homie so I think we should retain both. Or at a minimum have a long term where the original MQTT config still work so we don't break users configs in the short run but have a warning that it's going away. In either case, indeed we need to fix this if both do not work.

Reply:
When a channel is specified, it always works no matter whether you want to use Homie or not. 

But without defining the channel, at least the normal MQTT is broken.